chainstate/tx-verifier/src/transaction_verifier/input_output_policy.rs

```diff
@@ -311,6 +443,23 @@ mod tests {
         )
     }
 
+    fn create_delegation() -> TxOutput {
+        TxOutput::CreateDelegationId(Destination::AnyoneCanSpend, PoolId::new(H256::zero()))
+    }
+
+    fn delegate_staking() -> TxOutput {
+        TxOutput::DelegateStaking(Amount::ZERO, DelegationId::new(H256::zero()))
+    }
```

Comment:
What do these `H256::zero()` mean here? If it's a valid value here, maybe a special type would be appropriate here, something that behaves like `Option` but the `None` value is internally represented by `H256::zero`?

Reply:
The actual values don't matter here so I just stubbed them with the simplest. IO policy only cares about types combinations and not values inside